prometheus/go_collector.go

```diff
@@ -16,16 +19,31 @@ func NewGoCollector() *goCollector {
 			Name: "process_goroutines",
 			Help: "Number of goroutines that currently exist.",
 		}),
+		gcDesc: NewDesc(
+			"process_gc_duration_seconds",
```

Comment:
As per https://github.com/prometheus/client_golang/pull/113#issuecomment-98316261, this should be `process_go_gc_duration_seconds`.